src/Model/CustomerSegmentInterface.php

```diff
@@ -84,24 +80,19 @@ public function getUseAsTargetGroup(): ?bool;
     public function getTargetGroup(): ?string;
 
     /**
-     * @param string $targetGroup
+     * @param string|null $targetGroup
      *
      * @return $this
      */
     public function setTargetGroup(?string $targetGroup);
 
     /**
-     * @return []
+     * @return array
      */
     public function getDataForWebserviceExport();
 
     /**
-     * @return void
-     */
-    public function save();
-
-    /**
-     * @return void
+     * @return string
      */
-    public function delete();
+    public function __toString();
```

Comment:
ElementInterface have __toString since Pimcore X. I tried to add it via stub, but this doesn't work. I add a TODO comment so we can removed it when we drop Pimcore 6.9 https://github.com/pimcore/customer-data-framework/pull/248/commits/6aff9c524cb69ec0ec41d06355d165de965b925f